Meta/run.sh

```diff
@@ -245,6 +245,10 @@ fi
 
 if [ -z "$SERENITY_DISABLE_GDB_SOCKET" ]; then
     SERENITY_EXTRA_QEMU_ARGS="$SERENITY_EXTRA_QEMU_ARGS -gdb tcp:${SERENITY_HOST_IP}:1234"
+
+    if [ -n "$SERENITY_WAIT_FOR_GDB" ]; then
+        SERENITY_EXTRA_QEMU_ARGS="$SERENITY_EXTRA_QEMU_ARGS -S"
```

Comment:
(Kind of feels like there must be a way to do that already; the intel port would've needed this too (?))

(When I did super early bringup, IIRC I did something like `serenity.sh gdb aarch64` and that automatically did the right thing somehow maybe? It's been a while so I don't fully remember it, but there was some script that started gdb and stopped at the first instruction. Maybe it didn't run in qemu though?)

Reply:
Yeah, I saw that indeed, but IIRC that requires `tmux`, which for whatever reason was not installed on my PC, and I'm also not very familiar with it. So, I used different tabs in a terminal to do the same thing, and added this patch as it was easier then adding `-S` every time to `run.sh`.

I will remove this commit, and just keep it locally :)